internal/resource/resource.go

```diff
@@ -314,7 +313,7 @@ func CompareEnoManagedFields(a, b []metav1.ManagedFieldsEntry) bool {
 	if ai == -1 || ab == -1 {
 		return false
 	}
-	return reflect.DeepEqual(a[ai], b[ab])
+	return equality.Semantic.DeepEqual(a[ai], b[ab])
```

Comment:
Unrelated: just seems like a good idea since the managedFields struct contains timestamps